sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -50,14 +50,8 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  #commit b38855ed815ffd076bfde5e5e06170ca4f723dc1
-  #Merge: e6343f4 6a2c548
-  #Author: Piotr Balcer <piotr.balcer@intel.com>
-  #Date:   Thu Oct 5 12:15:42 2023 +0200
-  #    Merge pull request #920 from jsji/localcopy
-  #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/al42and/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG "ext_oneapi_queue_priority-hip")
```

Comment:
why is this not merged to [regular ](https://github.com/oneapi-src/unified-runtime.git)?

Reply:
This is a draft PR per the contribution guide of unified-runtime, for https://github.com/oneapi-src/unified-runtime/pull/984

Reply:
ok, so it's for testing it only not to merge

Reply:
Yes. Should I have indicated it differently?

Reply:
Perhaps add this info in the description and link to the UR PR